{"name": "Проверка невозможности принять заказ при некорректном ID курьера", "status": "passed", "steps": [{"name": "Создаём новый заказ и получаем его ID по трек-номеру", "status": "passed", "steps": [{"name": "Создаём новый заказ", "status": "passed", "parameters": [{"name": "color", "value": "None"}], "start": 1733069183643, "stop": 1733069183697}], "parameters": [{"name": "color", "value": "None"}], "start": 1733069183643, "stop": 1733069183842}, {"name": "Принимаем заказ по его ID и ID курьера с произвольными данными", "status": "passed", "parameters": [{"name": "order_id", "value": "385039"}, {"name": "courier_id", "value": "1000000"}], "start": 1733069183842, "stop": 1733069183899}], "start": 1733069183643, "stop": 1733069183900, "uuid": "d14cffb3-5697-48c5-b614-07bebc40b2c9", "historyId": "2187d1c5696e9555d5238263e2dc35ad", "testCaseId": "2187d1c5696e9555d5238263e2dc35ad", "fullName": "tests.test_take_order.TestTakeOrder#test_take_order_uncorrect_courier_id_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "DESKTOP-4GFTOQP"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_take_order"}]}